.github/workflows/test-rails.yaml

```
@@ -79,6 +84,12 @@ jobs:
            ref: deployed-to-production
            path: vendor/govuk-content-schemas
 
+       - name: Install additional system dependenices
+         if: ${{ inputs.extraSystemDependencies }}
+         run: |
+           sudo apt-get update
+           sudo apt-get install -y ${{ inputs.extraSystemDependencies }}
```

`--no-install-recommends` is good practice here to avoid installing unnecessary packages.
```suggestion
           sudo apt-get install -y --no-install-recommends ${{ inputs.extraSystemDependencies }}
```